lib/tasks/reimport.rake

```diff
@@ -0,0 +1,11 @@
+namespace :reimport do
+  desc 'Reimport TPI data - USE WITH CAUTION'
+  task tpi: :environment do
+    Company.delete_all
```

Comment:
maybe we can either:
- add required rake task param (like `rake reimport:tpi confirm=true)` in order to actually invoke it; without param it could print info what would be done
- or.. make it interactive, show `R u sure (y/n)` prompt - it is meant to be run manually anyway, right?

I can imagine someone will run it without reading description.

Reply:
I think that's not necessary, access to production is limited.